### PR TITLE
Add VIA support for LazyDesigners Dimple.

### PR DESCRIPTION
**Pull request by Erovia**
## Description
Add VIA support for LazyDesigners Dimple.
Assign a unique VID to all of their boards. (PID was already unique among themselves)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by Erovia** [comment]
A user keymap fails, but that's unrelated to this PR.

File: keyboards/lazydesigners/dimple/config.h [hunk 1]
```diff
@@ -20,7 +20,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
+#define VENDOR_ID       0x4C44
```
Comment (fauxpark) [inline keyboards/lazydesigners/dimple/config.h]:
```suggestion
#define VENDOR_ID       0x4C44 // "LD"
```

File: keyboards/lazydesigners/the30/config.h [hunk 2]
```diff
@@ -20,7 +20,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
+#define VENDOR_ID       0x4C44
```
Comment (fauxpark) [inline keyboards/lazydesigners/the30/config.h]:
```suggestion
#define VENDOR_ID       0x4C44 // "LD"
```

File: keyboards/lazydesigners/the50/config.h [hunk 3]
```diff
@@ -3,7 +3,7 @@
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
+#define VENDOR_ID       0x4C44
```
Comment (fauxpark) [inline keyboards/lazydesigners/the50/config.h]:
```suggestion
#define VENDOR_ID       0x4C44 // "LD"
```

File: keyboards/lazydesigners/the60/config.h [hunk 4]
```diff
@@ -3,7 +3,7 @@
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
+#define VENDOR_ID       0x4C44
```
Comment (fauxpark) [inline keyboards/lazydesigners/the60/config.h]:
```suggestion
#define VENDOR_ID       0x4C44 // "LD"
```

File: keyboards/lazydesigners/dimple/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+	[3] = LAYOUT(KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO)
+};
+
+void led_set_user(uint8_t usb_led) {
+if (IS_LED_ON(usb_led, USB_LED_CAPS_LOCK)) {
+	dimple_led_on();
+	} else {
+	dimple_led_off();
+	}
+}
```
Comment (fauxpark) [inline keyboards/lazydesigners/dimple/keymaps/via/keymap.c]:
```suggestion
void led_update_user(led_t led_state) {
    if (led_state.caps_lock) {
        dimple_led_on();
    } else {
        dimple_led_off();
    }
    return false;
}
```

**Pull request merged**